test/integ_tests/test_pulse.py

```diff
@@ -184,7 +184,10 @@ def cz_pulse(
 
 
 def test_pulse_bell(arbitrary_waveform, device):
-    a, b, = (
+    (
+        a,
+        b,
+    ) = (
```

Comment:
This feels weird to me? Did our linter change? (not crucial)

Reply:
Yeah, it was weird. I even did an update and still got an error. I also tried pushing the changes without this and the github actions failed. It finally defeated me and I pushed the linter changes too.